apps/system/js/nfc_handover_manager.js

```diff
@@ -249,9 +196,8 @@
       if (!this.isHandoverInProgress() &&
           Service.query('BluetoothTransfer.isSendFileQueueEmpty')) {
         if (this.bluetoothAutoEnabled) {
-          this._debug('Disabling Bluetooth');
-          // XXX: Let BluetoothCore do this for you
-          this.writeSetting({'bluetooth.enabled': false});
+          this.debug('Disabling Bluetooth');
+          window.dispatchEvent(new CustomEvent('request-disable-bluetooth'));
```

Comment:
Let's reuse this.publish from BaseModule here: this.publish('request-disable-bluetooth', this, /* no prefix */ true);